bitbot/links.go

```diff
@@ -26,8 +32,56 @@ func isURL(message string) bool {
 	return xurls.Strict.MatchString(message)
 }
 
+func wasLookedUpInTheLastMintues(url string) bool {
+	db, err := newDB()
```

Comment:
We already have a DB exposed to us as `b.DB`, so there is no need to create a new one.

Reply:
This should be a method attached to `Bot` so that it gets access to the shared DB instance spawned and available as `b.DB`